File: pattern/Pattern24.c
//////////////////////////////////////////////////////////////////////
//
// Accept number from user and display below pattern.
//                  
//
//////////////////////////////////////////////////////////////////////

/*
 iRow=4
  iCol=5
  
4 4 4 4 4
3 3 3 3 3
2 2 2 2 2
1 1 1 1 1

*/         

#include"Header.h"

int main()
{
  int iValue1=0,iValue2=0;
 
  printf("Enter the number of rows and columns\n");
  scanf("%d %d",&iValue1,&iValue2);

  Pattern24(iValue1,iValue2);
 

 return 0;
}

File: Java/Pattern/Pattern24.java
////////////////////////////////////////////////////////////////////////////////////
//
// Accept number of rows and number of columns  from user and display below pattern. 
//
///////////////////////////////////////////////////////////////////////////////////


/*Input : iRow = 6 iCol = 6 
Output : * * * * * * 
         * # # # * * 
         * # # * $ * 
         * # * $ $ * 
         * * $ $ $ * 
         * * * * * * 



*/




import java.util.*;



class Pattern
{
    public void Patterns(int iRows,int iCol)
    {
        int i=0,j=0;
        if(iRows!=iCol)
        {
            return;
        }
        for(i=1;i<=iRows;i++)
        {
            for(j=1;j<=iCol;j++)
            {
                if(i==1||i==iRows||j==1||j==iCol||j==iCol-i+1)
                {
                    System.out.print("*\t");

                }
                else if(j<iCol-i+1)
                {
                    System.out.print("#\t");
                }
                else
                {
                    System.out.print("$\t");
                }
               
               
            }
             System.out.println();
        }
    }
  
}
class Pattern24
{
    public static void main(String arg[])
    {
        int iRows=0,iCol=0;
        Scanner sc=new Scanner(System.in);
        iRows=sc.nextInt();
        iCol=sc.nextInt();
        Pattern pobj=new Pattern();
        pobj.Patterns(iRows,iCol);

    }
}